rcl/include/rcl/lexer_lookahead.h

```diff
@@ -31,13 +31,15 @@ extern "C"
 #endif
 
 // Forward declaration
-struct rcl_lexer_lookahead2_impl_t;
+struct rcl_lexer_lookahead2_impl_s;
+
+typedef struct rcl_lexer_lookahead2_impl_s rcl_lexer_lookahead2_impl_t;
```

Comment:
And here (though this one looks harder to remove, as the tests seem to use the impl structure).

Reply:
Same as above.